src/frontend/context/fagsak/FagsakContext.tsx

```diff
@@ -1,109 +1,52 @@
-import React, { createContext, type PropsWithChildren, useContext, useState } from 'react';
-
-import type { AxiosError } from 'axios';
-import deepEqual from 'deep-equal';
+import React, {
+    createContext,
+    type PropsWithChildren,
+    useContext,
+    useEffect,
+    useState,
+} from 'react';
 
 import { useHttp } from '@navikt/familie-http';
 import type { Ressurs } from '@navikt/familie-typer';
 import {
     byggDataRessurs,
-    byggFeiletRessurs,
     byggHenterRessurs,
     byggTomRessurs,
-    hentDataFraRessurs,
     RessursStatus,
 } from '@navikt/familie-typer';
 
-import { useOppdaterBrukerOgEksterneBehandlingerNårFagsakEndrerSeg } from './useOppdaterBrukerOgKlagebehandlingerNårFagsakEndrerSeg';
 import useFagsakApi from '../../api/useFagsakApi';
-import { useTilbakekrevingApi } from '../../api/useTilbakekrevingApi';
 import type { SkjemaBrevmottaker } from '../../sider/Fagsak/Fagsaklinje/Behandlingsmeny/LeggTilEllerFjernBrevmottakere/useBrevmottakerSkjema';
 import type { IMinimalFagsak } from '../../typer/fagsak';
-import type { IKlagebehandling } from '../../typer/klage';
 import type { IPersonInfo } from '../../typer/person';
-import type { ITilbakekrevingsbehandling } from '../../typer/tilbakekrevingsbehandling';
 import { sjekkTilgangTilPerson } from '../../utils/commons';
-import { obfuskerFagsak, obfuskerPersonInfo } from '../../utils/obfuskerData';
+import { obfuskerPersonInfo } from '../../utils/obfuskerData';
 import { useAppContext } from '../AppContext';
 
 interface IFagsakContext {
     bruker: Ressurs<IPersonInfo>;
-    hentMinimalFagsak: (fagsakId: string | number, påvirkerSystemLaster?: boolean) => void;
-    minimalFagsakRessurs: Ressurs<IMinimalFagsak>;
-    settMinimalFagsakRessurs: (fagsak: Ressurs<IMinimalFagsak>) => void;
-    minimalFagsak: IMinimalFagsak | undefined;
-    hentBruker: (personIdent: string) => void;
-    klagebehandlinger: IKlagebehandling[];
-    klageStatus: RessursStatus;
-    oppdaterKlagebehandlingerPåFagsak: () => void;
-    tilbakekrevingsbehandlinger: ITilbakekrevingsbehandling[];
-    tilbakekrevingStatus: RessursStatus;
+    minimalFagsak: IMinimalFagsak;
```

Comment:
Syns denne kan like greit hete `fagsak`, på sikt burde vi omdøpe `IMinimalFagsak` til `Fagsak` 

```suggestion
    fagsak: IMinimalFagsak;
```